e2e/cypress/integration/channel_sidebar/more_unreads_position_with_scroll_spec.js

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channel_sidebar
+
+import {getRandomId} from '../../utils';
+
+describe('Channel sidebar', () => {
+    let mainUser;
+    let otherUser;
+    let myTeam;
+
+    // This will ensure a list of channel names that will be alphabetically sorted
+    const channelNames = new Array(20).fill(1).map((value, index) => `scroll${index}`);
```

Comment:
A bit dirty I think but this is an easy way for me to ensure that I have a list of 20 elements that seems to be enough to have the scrollbar display in the channels list.

Reply:
See https://github.com/mattermost/mattermost-webapp/pull/6030#discussion_r463637425